### ISSUE TITLE
Can you add the Browser and OS information below the DIM version?

### ISSUE THREAD

**Issue by mannyrye** [DestinyItemManager/DIM#8589]
### Proposed change

Can you add the Browser and OS information below the DIM version?

### How does this fit into your workflow?

That would simplify bug submissions, and you'd be more likely to get the accurate information you want.

**Comment by bhollis** [comment]
I guess? I think most people know what browser and OS they use?

**Comment by mannyrye** [comment]
I use DIM on Safari, Chrome and Firefox on my Mac, Edge on my XBox, Safari, Chrome and App Store apps on my iPhone and iPad. I know what platform and browser I'm on, but I like to double check system versions when submitting bug reports and I usually don't know what browser version I'm on. 😀

**Comment by WalkerSakatch** [comment]
I figured I would take this as my first issue since it seems easy enough. Not sure where a good spot would be to put the info though. Do you guys think just here on the about page is fine or does somewhere else make more sense? 
![image](https://user-images.githubusercontent.com/4109476/183568128-9869266d-4771-43a9-9347-c193fed45c2f.png)

**Comment by bhollis** [comment]
Yeah, it could go right below the version on that page

**Comment by nev-r** [comment]
is this chrome or safari though

**Comment by WalkerSakatch** [comment]
That screenshot is from chrome. For some reason, on chromium based browsers it shows both chrome and safari in the printout. If the user were using Firefox for example it should only say Firefox. I could add some type of method to parse the value of userAgent if you feel is necessary

**Comment by bhollis** [comment]
Maybe we should use our user agent parser library

**Comment by WalkerSakatch** [comment]
Thanks, I didn't realize there was already a library installed to deal with this sort of thing. I updated the PR a bit. I'm importing the library that I hope you were talking about. I'm not really familiar with typescript, and my editor was yelling at me about not having types for that library installed, so I ran an npm command to install them, and it added to the package.json. Let me know if I shouldn't have committed that along with the actual change
